src/plugins/intel_cpu/tests/unit/snippets_transformations/x64/lowered/buffer_allocation.cpp

```diff
@@ -282,7 +270,7 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_BufferAllocation_MHANotOptimizedWSplit,
                          ::testing::Combine(
                                  ::testing::Values(false),
                                  ::testing::Values(true),
-                                 ::testing::Values(75264), // Each Buffer has own allocated memory
+                                 ::testing::Values(591360), // Each Buffer has own allocated memory
```

Comment:
I increased MNK dims in order to keep the loops markup the same as before. I assumed that if "reg group" and "clusters" counts don't change, than the correctness is not affected.
However, since you recommend to check the correctness manually, I checked the serialized graphs: bf16 test cases with and without SplitLoops concerned me (their comparison), so I would discuss this case offline with you.
The rest cases are correct, I have no concerns to them

Reply:
Wrote offline: `SplitLoops` gives benefit when dimension `M` is larger than `M_block_size`. Previously, there were `M=128` and `M_block_size=32`. Now `M=64` and `M_block_size=32`.
I think if we increase `M`, we will see `SplitLoops` benefits again.

If CI is green and no need to correct something else in this PR, we can merge so. However, I'd suggest to fix `M` in the follow-up PR